code-for-nashville/glean-tennessee-web
# updating jquery version to reflect improvements

Because the React migration is taking some time, and it would be good to get a working testable prototype for farmers to try out, I'm filling in some missing details in the jQuery version of the app.
On this PR: 
The glean request button hides until a person is logged in or registered. (#19, #21) and once they do log in, the register and log in links disappear from the nav (#15).

Log out functionality works and is now linked to the nav buttons. (#8, #21, #15).

Also, I've added the checkbox for is_organic as per #13. (More additions needed)

To Test:
1. Fetch the branch and pull it down.
1. Check the links available before signing in, and confirm what is available after signing in (or signing up).
1. Ensure that when logged in, the log out and glean submission buttons are available.
1. Click log out and ensure the log in and register buttons are now available and that the glean submit and log out buttons are gone. (#8, #15, #21)
1. register a new account and check the console log to see that there is an object reflecting the is_organic boolean. (If you checked the box, it should say true, if not, false)
